### DISCUSSION TITLE
About seepage and 3d-model, and program error

### DISCUSSION THREAD

**Discussion by qwerty12345697** [Anura3D/Anura3D_OpenSource#177] [Q&A]
As soon as I use 3D modeling, I will get an error as long as I add the hydraulic boundary conditions (seepage/head, etc.), delete the hydraulic conditions, and open the hydraulic boundary conditions in the calculation data, according to the tips discussed (simulate rainfall you can apply a zero pressure at the top surface (surface saturated)), does it mean that the hydraulic boundary conditions can not be added, If the seepage surface is not turned on in the calculation data, can it mean that the simulated rainfall has been successful?

Another crucial question is that there will be no errors in the calculation, but no matter how long I wait, the calculation will not be completed (this is more than when I used two different soil materials in the rainfall simulation), and when I debug it, it shows that the calculation is still in progress, I can't seem to go any further, can it help me.
<img width="863" alt="1" src="https://github.com/Anura3D/Anura3D_OpenSource/assets/57770605/098e0e21-b1a1-447b-8b86-7c3b4bb78547">
<img width="1232" alt="2" src="https://github.com/Anura3D/Anura3D_OpenSource/assets/57770605/946c216b-7510-4a5a-93c1-76bc54447ef9">

**Comment by FrancescaCeccato** [comment]
Unfortunately the hydraulic BC are not implemnted for 3D cases, this is why you cannot use them. However, if you want to simulate a heavy rainfall you can apply zero pressure at the same boundary.

Regarding the second question, your simulation is probably stuck somewhere, maybe a loop. You have to debug to find it. Usually the constitutive models create issues like this.

**Reply by DoHungThoi** [reply]
You can show the example to assign the zero pressure at the boundary in the model or you can make a video to guide for simulating this case?

**Reply by FrancescaCeccato** [reply]
This is very easy: do not apply any condition to the boundary! Indeed, we solve the velocity-velocity formulation, which means that zero traction or zero pressures at boundaries are automatic!

**Reply by jiawei6688** [reply]
Thank you for sharing. Is this issue still unresolved in Anura3D version 2025?
When I was doing the step-by-step calculations, I still couldn't calculate the stage with the hydraulic conditions set.

**Comment by DoHungThoi** [comment]
![Screenshot 2024-10-21 200228](https://github.com/user-attachments/assets/ac17b075-016d-4a61-ac16-3fd4b5d7ebd0)
How simulate rainfall-induced landslide that give the result as the paper?

**Comment by DoHungThoi** [comment]
Dear Francesca Ceccato, For modelling the fully saturated soil at the boundary, zero pore pressure is applied to the material points located in the surface elements. Can you tell me how to apply the zero pore pressure to the material points in the surface elements?

**Reply by FrancescaCeccato** [reply]
Zero pressure is automatic, just leave no boundary conditions at that boundary.

**Reply by DoHungThoi** [reply]
![Screenshot 2024-10-22 190001](https://github.com/user-attachments/assets/e030c3c1-b8d2-4c42-ae84-863b491f7c82)
Dear Francesca Ceccato. As you see in the picture, this the way that apply zero pore pressure to  the material points located in the surface elements to model continuous water infiltration into the slope

**Reply by FrancescaCeccato** [reply]
This is solid traction, You need to select pore pressure.

**Comment by DoHungThoi** [comment]
![Screenshot 2024-10-22 212604](https://github.com/user-attachments/assets/1d3eea76-0d2c-4508-a081-b9fadebc3521)
![Screenshot 2024-10-22 212620](https://github.com/user-attachments/assets/3104d489-f7a0-475d-b8f9-5feaaae22d30)
![Screenshot 2024-10-22 212646](https://github.com/user-attachments/assets/fbf2a982-577b-4fa9-a558-571c11c50006)
![Screenshot 2024-10-22 212653](https://github.com/user-attachments/assets/9037294e-4d48-4da0-87b1-45a7f09f13c6)
Dear Francesca Ceccato. According to simulate the rainfall-induced the slope collapse that i will set up into Calculation data. Is it okay, Francesca Ceccato?

**Comment by DoHungThoi** [comment]
![Screenshot 2024-10-23 090722](https://github.com/user-attachments/assets/8dd78cc9-f837-422f-ab54-3405685bf897)
I already set up the gravity loading = 50 kPa as the above picture. But I do not know that is exactly.
Dear Francesca Ceccato. You can guide how to set up in Anura3D: "gravity loading to the generate the in-situ stresses and application of an initial prescribed suction of 50 kPa across the domain and explain it.

**Reply by FrancescaCeccato** [reply]
First of all, why are you using anura3D 2022? You should move to version 2024!
Anyway...The prescribed suction is wrong, because this is a solid traction and not a pressure. On the top you need to choose "Liquid pressure".
K0 procedure is not applicable to this case, at least with horizontal surface, you should consider a not-horizontal surface or do gravity loading without K0.

**Reply by DoHungThoi** [reply]
![1](https://github.com/user-attachments/assets/9962e9d4-7d4c-4759-9c92-e59cb1d9c77d)
![2](https://github.com/user-attachments/assets/2251a9e6-d100-431c-9f34-5dc3c7cb961c)
Dear Francesca Ceccato, I also tried to use the GiD17.0.1-win-x64-Install and Anura3D-OpenSource-main. But I can not run the model as the pictures. You can help me and explain how to solve this problems?

**Reply by FrancescaCeccato** [reply]
The executable is misisng in your folder. Please read carefully the instructions in the first chapters of the tutorial manual.

**Reply by DoHungThoi** [reply]
![Screenshot 2024-10-23 165135](https://github.com/user-attachments/assets/dde2f4a8-c1ca-4258-9c32-2291bb407300)
The tutorial manual is error, Francesca Ceccato!

**Reply by FrancescaCeccato** [reply]
It can be downloaded without problems!
![image](https://github.com/user-attachments/assets/5d2ea074-7a0b-4b22-b882-762b9ac0a4bb)

**Reply by FrancescaCeccato** [reply]
Click to "download raw file" it is only a visualization issue of Github.

**Reply by DoHungThoi** [reply]
![Screenshot 2024-10-25 154405](https://github.com/user-attachments/assets/59e49f5f-9604-42f2-9e1d-62e6b9020f79)
![Screenshot 2024-10-25 154420](https://github.com/user-attachments/assets/eef046ec-5868-47ee-b77f-a9145b40cbcd)
![Screenshot 2024-10-25 154429](https://github.com/user-attachments/assets/3db44a2b-01df-4988-8f96-4c5269cb1d26)
Dear Francesca Ceccato, I saw the different between Anura3D 2024, 2023 and 2022 as the above picture. But when I used the Anura 3D 2023, the problem is occurred as the below picture. Can you explain for me? How to solve these problems! Thanks you!
![Screenshot 2024-10-25 155010](https://github.com/user-attachments/assets/05c26ba4-ca6d-4d94-8eda-f23100107ed2)

**Reply by FrancescaCeccato** [reply]
Did you convert the project to the new problemtype? In this case there may be small erros, and you need to check that all assigments are correct (materials, boundary conditions etc). Then generate the mesh at the Anura3D input files.

**Comment by DoHungThoi** [comment]
![Screenshot 2024-10-25 162017](https://github.com/user-attachments/assets/ed19b1fd-f62b-4d47-a37f-44daa5650e32)
Dear Francesca Ceccato, I also reached of  the Anura3D 2024. But the problem in the newest version that i can not do that. Can you help me?

**Reply by FrancescaCeccato** [reply]
Did you put Anura3D_2024.exe in the project folder? This messagfe is telling you that the executable is not there!

**Reply by DoHungThoi** [reply]
Dear FrancescaCeccato, I also do follow the Tutorial Manual but The model can still not run. I do not still understand the way that i can solve this problem.

**Reply by FrancescaCeccato** [reply]
The above message indicates that the "calculate.bat" file is trying to run the code "Anura3D_2024.exe", but this is not available. Make sure that the execuable is in the project folder. Try to check the spelling of the address. You can check by doing a rigth click on the calcualte.bat file and select "Modify".

**Reply by DoHungThoi** [reply]
![Screenshot 2024-11-07 174154](https://github.com/user-attachments/assets/8fa3b12f-55da-4821-a3de-f60caf881b61)
Dear FrancescaCeccato, I already put the Anura3D into the folder and built the model. But it not working

**Reply by DoHungThoi** [reply]
![Screenshot 2024-11-07 180757](https://github.com/user-attachments/assets/546c8052-cb60-4a24-8445-bc65b89536b9)
This the Generate Anura3D 2024 that I already created. But, i see it is very strange with the Generate Anura3D 2022.

**Comment by DoHungThoi** [comment]
![Screenshot 2024-10-28 162439](https://github.com/user-attachments/assets/1d76b88e-deff-47c9-9870-7ca17c1b01e3)
Dear FrancescaCeccato, You can help me to assign the condition for the model.

**Reply by FrancescaCeccato** [reply]
it is not very clear. I don't understand if 50kPa are uniform though the slope or applied only at the surface. 
If it is only at the surface, which is more reasonable, it means that a liquid pressure of 50kPa is applied initially at the slope surfce and then removed. You can do this with the load multipliers for the liquid. Initially the load multiplier will be 1, then you decrease it to 0 for the rainfall infiltration.

**Comment by DoHungThoi** [comment]
![Screenshot 2024-11-07 215601](https://github.com/user-attachments/assets/aac18a2d-38af-486d-97fe-7ff7a62694d1)
Dear Francesca Ceccato, You can guide me to built the model as the picture. I do not know how to set-up the background into squares size 1.0m x 1.0m. A total of 440 4 - node quadrilateral elements is used for the background mesh.

**Comment by jiawei6688** [comment]
mark down.
Thank you for sharing. This issue has not yet been resolved in Anura3D version 2025.

**Comment by DoHungThoi** [comment]
How can I compile the soft soil model (SSM) and soft soil creep model
(SSCM) in Anura3D 2025?

Vào Thứ 6, 16 thg 1, 2026 vào lúc 15:07 jiawei6688 <
***@***.***> đã viết:

**Reply by hjk1108** [reply]
Can you add other constitutive models now?

**Comment by DoHungThoi** [comment]
I am finding out the way to generate the soft soil creep constitutive model
and soft soil constitutive model in Anura3D/Anura3D_OpenSource Code.

Vào Thứ 2, 23 thg 3, 2026 vào lúc 15:33 hjk1108 ***@***.***>
đã viết:

**Reply by hjk1108** [reply]
I don't have any ideas at all now, do you have it now？

**Comment by DoHungThoi** [comment]
No, I am studying it, but it is very difficult. You can share with me the
materials about this problem.

Vào Thứ 4, 25 thg 3, 2026 vào lúc 22:03 hjk1108 ***@***.***>
đã viết:
